src/Types.hs

```diff
@@ -262,3 +273,10 @@ mailTags = lens _mailTags (\m t -> m { _mailTags = t })
 
 mailId :: Lens' NotmuchMail ByteString
 mailId = lens _mailId (\m i -> m { _mailId = i })
+
+class Scrollable n where
+  makeViewportScroller :: AppState -> ViewportScroll n
+
+-- XXX this is currently our only scrolling view
```

Comment:
the upcoming "list of key bindings" i.e. "help" view will be scrollable too, I guess

Reply:
Yes it will. Tho, isn't this single instance for `Scrollable n` pointless?